js/widgets/site-search.js

```diff
@@ -6,9 +6,9 @@ define([
     'dataset/typeahead',
     'reuse/typeahead',
     'organization/typeahead',
-    // 'user/typeahead',
+    'territory/typeahead',
```

Comment:
Here we're hitting the server for territories’ suggestions even if the feature is deactivated.